snuba/datasets/storages/events.py

```diff
@@ -88,6 +93,7 @@ def events_migrations(
         # optional stream related data
         ("offset", Nullable(UInt(64))),
         ("partition", Nullable(UInt(16))),
+        ("message_timestamp", Nullable(DateTime())),
```

Comment:
This is defined as `Nullable` to stay consistent with the related columns in this dataset, but does make it inconsistent with the other `message_timestamp` types. I don't have a strong preference one way or another.

Reply:
none either. This table is supposed to disappear anyway.

Reply:
Let's make it non-nullable like the rest since nullables add yet another two files to create/delete on write.

Reply:
Done, 796dac06da6bada15a6bea14c91f8ecc5441cad6.